source/technical/images.rst

```diff
@@ -207,6 +207,42 @@ You may view details of an image with the command:
 
 Replace ``image`` with either an image name or it's *UUID*.
 
+Sharing an Image
+________________
+
+You may share images several ways.  If you wish to share an image with everyone, use:
+
+.. code-block:: bash
+
+   openstack image set --public <image_id>
+
+Replace ``image`` with the image *UUID*.
```

Comment:
change `image` to `image_id` or modify the command line (`image_id` -> `image`) to keep consistent.